tools/experiments/README.md

```diff
@@ -38,11 +38,29 @@ Document can choose to opt in a certain experiments. To do that, simply put a me
 ```
 
 By doing so, the specified experiments will be enabled for all visitors of the document.
-Note not all experiments allow document level opt-in. For a full list of white listed experiments, please see `allow-doc-opt-in` attribute in the project's `prod-config.json` file.
-Note that document opt-in can be overridden by user opt-out.
+
+Note that **only some experiments** allow document level opt-in. For a full list of allowed experiments, please see `allow-doc-opt-in` attribute in [the `prod-config.json` file.](../../build-system/global-configs/prod-config.json)
```

Comment:
Can we change this one to:

`Note: **only some experiements**....`

This syntax will show it as notecard on amp.dev